### PR TITLE
add orientation change btn for 2 vert and 2 hort

### PR DESCRIPTION
**Pull request by gtodd876**

### CODE REVIEW

**Comment by FPGSchiba** [comment]
Ok so it is working for the 2 Vertical and 2 Horizontal...
We do have a mechanism within the MainWindow to change the Panels. Could we use this mechanism to change the Panels so we have all Code relating to closing and opening in one place?

If you want we can talk about that at sunday 😄

**Comment by FPGSchiba** [comment]
This looks good to me 😄 
Will you make this for all the Overlays on this branch or just the 2s right now?

**Review by FPGSchiba — APPROVED** [review]

**Review by FPGSchiba — APPROVED** [review]
Looks good to me, if you are ready you can merge those changes

File: DCS-SR-Client/UI/RadioOverlayWindow/RadioOverlayThreeHorizontal.xaml [hunk 1]
```diff
@@ -47,15 +47,13 @@
...
-                       Width="370"
-                       Height="16"
-                       Margin="5,0,0,0"
-                       VerticalAlignment="Center"
-                       FontSize="13"
-                       Foreground="#E7E7E7"
-                       Padding="0"
-                       Text="3 Radio Panel Horizontal" />
+            <TextBlock x:Name="ControlText"
+           Width="310"
```
Comment (Dabble1SC) [inline DCS-SR-Client/UI/RadioOverlayWindow/RadioOverlayThreeHorizontal.xaml:51]: additional tabbing needed for the control text section

**Review by Dabble1SC — COMMENTED** [review]
Looks good.

**Pull request merged**